techmatters/flex-plugins
# View Contact/Case Permissions

## Description
This PR adds the following permissions:
- `viewContact`
- `viewCase`

The counselor should never be able to view a Contact or Case if he/she doesn't have permission for it.

### Related Issues
Fixes https://tech-matters.atlassian.net/browse/CHI-1642
Backend PR: https://github.com/techmatters/hrm/pull/316

### Verification steps
If the backend PR is not merged:
- Set `viewContact` permission to `[["isSupervisor"], ["isOwner"]]`
- Search for contacts
- Check that you cannot view details of contacts you don't own
- Check that you can view details of contacts you own
- Repeat the same for Cases

If the backend PR is merged:
- Set `viewContact` permission to `[["isSupervisor"], ["isOwner"]]`
- Search for contacts
- All the contacts displayed are the ones you own
- Check that you can view their details
- Repeat the same for Cases
